src/actions/Dashboard/index.js

```diff
@@ -118,7 +113,16 @@ const methods = {
         });
     },
     changeLanguage(language) {
-        this.dispatch(constants.DASHBOARD.CHANGE_LANGUAGE, language);
+        AdminServices.getWatchlist(language, (error, response, body) => {
+            if(!error && response.statusCode === 200 && body.data && !body.errors) {
+                const { terms } = body.data;
+
+                this.dispatch(constants.DASHBOARD.CHANGE_LANGUAGE, { language, terms });
+            } else {
+                console.error(error, null);
+                self.dispatch(constants.DASHBOARD.LOAD_DETAIL_ERROR, error);
```

Comment:
`self` here is undefined which is causing a lint issue:

![image](https://user-images.githubusercontent.com/1086421/30418102-23fe0a56-9932-11e7-8f2a-2a495e8f937f.png)

Easy fix: just declare `const self = this` between lines 115-116 above.